ios/RNCWKWebView.m

```diff
@@ -142,6 +142,69 @@ - (void)didMoveToWindow
     wkWebViewConfig.mediaPlaybackRequiresUserAction = _mediaPlaybackRequiresUserAction;
 #endif
 
+    if(_sharedCookiesEnabled) {
+      // More info to sending cookies with WKWebView
+      // https://stackoverflow.com/questions/26573137/can-i-set-the-cookies-to-be-used-by-a-wkwebview/26577303#26577303
+      if (@available(iOS 11.0, *)) {
+        // Set Cookies in iOS 11 and above, initialize websiteDataStore before setting cookies
+        // See also https://forums.developer.apple.com/thread/97194
+        wkWebViewConfig.websiteDataStore = [WKWebsiteDataStore nonPersistentDataStore];
```

Comment:
This is already defined by cacheEnabled props, are you sure this still belongs here?